model/user.go

```diff
@@ -80,6 +80,7 @@ type User struct {
 	MfaActive          bool      `json:"mfa_active,omitempty"`
 	MfaSecret          string    `json:"mfa_secret,omitempty"`
 	LastActivityAt     int64     `db:"-" json:"last_activity_at,omitempty"`
+	IsBot              bool      `db:"-" json:"is_bot"`
```

Comment:
Maybe use `omitempty`?

Reply:
It will never be empty.

Reply:
If `IsBot` is `false` and you set `omitempty`, it will not be rendered in JSON. See https://play.golang.org/p/Q4roL9DRwXl for am example.

Reply:
That makes sense. Fixed.